test/e2e-common.sh

```diff
@@ -60,9 +60,16 @@ readonly BUCKETS=10
 LATEST_SERVING_RELEASE_VERSION=$(latest_version)
 
 # Latest net-istio release.
-LATEST_NET_ISTIO_RELEASE_VERSION=$(
-  curl -L --silent "https://api.github.com/repos/knative/net-istio/releases" | \
-    jq -r '[.[].tag_name] | sort_by( sub("knative-";"") | sub("v";"") | split(".") | map(tonumber) ) | reverse[0]')
+LATEST_NET_ISTIO_RELEASE_VERSION=$(align_net-istio_version "$LATEST_SERVING_RELEASE_VERSION")
+
+# Receives the latest serving version and searches for the same version with major and minor and searches for the latest patch
+function align_net-istio_version() {
+  local serving_version=$1
+  local major_minor
+  major_minor=$(echo "$serving_version" | cut -d '.' -f 1,2)
+
+  curl -L --silent "https://api.github.com/repos/knative/net-istio/releases" | jq --arg major_minor "$major_minor" -r '[.[].tag_name] | map(select(. | startswith($major_minor))) | sort_by( sub("knative-";"") | sub("v";"") | split(".") | map(tonumber) ) | reverse[0]'
```

Comment:
the net-istio repo will have prefixes with `knative-`. The `map(select(. | startswith($major_minor)))` stage will need to drop that prefix or change `startswith` to `contains` maybe?

Reply:
see comment above